cmd/tink-worker/internal/registry.go

```diff
@@ -46,8 +56,12 @@ func (r *RegistryConnDetails) NewClient() (*client.Client, error) {
 	return c, nil
 }
 
+type ImagePuller interface {
```

Comment:
Does this interface need to be exported?

Reply:
done.